storage/s3_sdk/list_gcs_buckets.go

```diff
@@ -42,15 +45,15 @@ func listGCSBuckets(w io.Writer, googleAccessKeyID string, googleAccessKeySecret
 
 	result, err := client.ListBucketsWithContext(ctx, &s3.ListBucketsInput{})
 	if err != nil {
-		return nil, err
+		return err
```

Comment:
Mind adding context to the error?

```
return fmt.Errorf("ListBucketsWithContext: %v", err)
```

That way, you know where the errors came from.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors